crates/task-impls/src/quorum_vote/handlers.rs

```diff
@@ -298,6 +298,7 @@ pub(crate) async fn submit_vote<TYPES: NodeType, I: NodeImplementation<TYPES>, V
     storage: Arc<RwLock<I::Storage>>,
     leaf: Leaf<TYPES>,
     vid_share: Proposal<TYPES, VidDisperseShare<TYPES>>,
+    extended_vote: bool,
```

Comment:
I'm not sure I understand the reason for this either -- are we planning for the `submit_vote` to actually be different for an eqc vs a regular qc? it seems like the only difference is we create a different internal hotshot event (which gets broadcast in exactly the same way)